SoftLayer/CLI/deprecated.py

```diff
@@ -0,0 +1,17 @@
+"""
+    SoftLayer.CLI.deprecated
+    ~~~~~~~~~~~~~~~~~~~~~~~~
+    Handles usage of the deprecated command name, 'sl'.
+    :license: MIT, see LICENSE for more details.
+"""
+import sys
+
+import click
+
+
+@click.command()
```

Comment:
I think making this a click command is a mistake since it will error out before showing the message in most cases.